src/scripts/release_picard.sh

```diff
@@ -184,12 +184,12 @@ cd $TMPDIR
 # clone
 git clone $PICARDGITROOT picard
 cd picard
-ant clone-htsjdk
+ant -buildfile build-from-src.xml clone-htsjdk
```

Comment:
Ok, sounds good -- I'll create a separate ticket to track the release process changes.